sycl/include/sycl/detail/vector_convert.hpp

```diff
@@ -57,6 +57,9 @@
 #include <sycl/detail/generic_type_traits.hpp> // for is_sigeninteger, is_s...
 #include <sycl/exception.hpp>                  // for errc
 
+#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
+#include <sycl/detail/memcpy.hpp> // for memcpy
```

Comment:
Nit; We don't add the comments anymore.
```suggestion
#include <sycl/detail/memcpy.hpp>
```